### PR TITLE
Setups up Winston + Elastic + Kibana and starts tests

### PR DESCRIPTION
**Pull request by JamesTheTerry**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,21 +11,24 @@
...
-    "random-date-generator": "^1.0.2"
+    "random-date-generator": "^1.0.2",
+    "supertest": "^3.0.0"
   },
   "engines": {
     "node": ">=8.0.0"
   },
   "scripts": {
-    "start": "node --max-old-space-size=8192 server",
+    "start": "node server-unused",
```
Comment (dustinwhuang) [inline package.json]: Is this still needed?

File: micro-consumer/winston/winston.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+var winston = require('winston');
```
Comment (dustinwhuang) [inline micro-consumer/winston/winston.js]: Consider using const for variables that don't need reassignment

File: micro-consumer/winston/winston.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+var Elasticsearch = require('winston-elasticsearch');
+var eSearch=require('elasticsearch');
```
Comment (dustinwhuang) [inline micro-consumer/winston/winston.js]: Spacing on equals

File: package.json (hunk 1 above)
Comment (JamesTheTerry) [reply package.json]: No

File: micro-consumer/winston/winston.js (hunk 2 above)
Comment (JamesTheTerry) [reply micro-consumer/winston/winston.js]: Thanks I miss that some times

**Pull request merged**